src/rules/arrayBracketSpacingRule.ts

```diff
@@ -19,7 +19,7 @@ export class Rule extends Lint.Rules.AbstractRule {
       - \`"always"\`requires one or more spaces or newlines inside array brackets
 
       The second option is an object for exceptions to the \`"never"\` option:
-      
+
```

Comment:
There are trailing whitespace errors inside templates in more files. Did you run the linter?